docs/api/hardhat/reference.md

```diff
@@ -24,30 +24,32 @@ This plugin most often will not be used directly in the code.
 
 ### Configuration
 
-```js
+```json
```

Comment:
why?

The code below is not a valid json.

Reply:
Also not a valid javascript though. I can make it a valid one if you have a strong preference, but it looks ok either way.